packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -36,7 +42,9 @@ export default function ShareModal({
     ? `${post?.commentsPermalink}${getCommentHash(comment.id)}`
     : post?.commentsPermalink;
   const { trackEvent } = useContext(AnalyticsContext);
-  const [, copyUrl] = useCopyLink(() => link);
+  const [commentary, setCommentary] = useState<string>('');
```

Comment:
You don't need to typecast <string> if you pass a empty string directly.
```suggestion
  const [commentary, setCommentary] = useState('');
```

Reply:
@sshanzel Just a note this shouldn't be needed right?

Reply:
Ah, right. My bad, let me update.